### DISCUSSION TITLE
Nugram::mixin(new CustomMethods()); - error - Undefined method ...

### DISCUSSION THREAD

**Discussion by ghost** [nutgram/nutgram#322] [Q&A]
Hi.

![Screenshot from 2023-02-13 14-16-32](https://user-images.githubusercontent.com/78247242/218437809-2ae0454a-ac69-4ce6-a52b-40a4553ee795.png)

My Code is working but how can I make these errors not appear in VSCode?

**Comment by Lukasss93** [comment]
```php
/* @var Nutgram $this */
```

**Reply by ghost** [reply]
Where exactly should I put this it in the code?
No matter where I put it (above the class, above the method, etc.), the problem is not solved :/

**Reply by Lukasss93** [reply]
```php
//...
return function($message) {
    /* @var Nutgram $this */
//...
```

**Reply by ghost** [reply]
It still doesn't work !

![Screenshot from 2023-02-13 14-33-16](https://user-images.githubusercontent.com/78247242/218441233-8610ee03-fbcd-414e-8ba0-302923b88372.png)

I closed vscode once and opened it again, but the problem is still there :/
Ignore it. It's not important. I will leave it as it is.
Delete this discussion if you want.
Thank you anyway :)
